src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -10976,52 +11335,41 @@ namespace HVACUnitarySystem {
 		// The air flow rate in cooling, heating, and no cooling or heating can be dIFferent.
 		// Calculate the air flow rate based on initializations.
 
-		// REFERENCES:
-		// na
-
 		// Using/Aliasing
 		using namespace DataZoneEnergyDemands;
 
 		// Locals
-		// SUBROUTINE ARGUMENT DEFINITIONS:
-
-		// SUBROUTINE PARAMETER DEFINITIONS:
-		// na
-
-		// INTERFACE BLOCK SPECIFICATIONS
-		// na
-
-		// DERIVED TYPE DEFINITIONS
-		// na
-
 		// SUBROUTINE LOCAL VARIABLE DECLARATIONS:
 		int InletNode; // inlet node number
 		Real64 AverageUnitMassFlow; // average supply air mass flow rate over time step
 		int SpeedNum; // speed for multi-speed or variable-speed coils
 		bool FanOn;
+		Real64 fanPartLoadRatio; // local variable for PLR
 
```

Comment:
here I introduced a local fanPLR variable so I could bait and switch when the ASHRAE model was active. The fan PLR and coil PLR are independent. Maybe a better way to do this, but this is what's here now.